src/frontend/src/defaults/formatters.tsx

```diff
@@ -117,7 +117,23 @@ export function formatPriceRange(
   )}`;
 }
 
-interface RenderDateOptionsInterface {
+/*
+ * Format a file size (in bytes) into a human-readable format
+ */
+export function formatFileSize(size: number) {
+  const suffixes: string[] = ['B', 'KB', 'MB', 'GB'];
```

Comment:
Comment (sarcastic, no change needed): Let's get ready for the gibibyte discussion